CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Fixed a segmentation violation issue when running Apptainer checkpoint.
 - Added apparmor profiles for ubuntu 24.04 or higher distros.
 - Fixed an issue that Apptainer won't read default docker credentials.
+- Trial fix when starting apptainer failed with `instance --fakeroot`.
```

Comment:
The message needs to be in the Changes for v1.3.x section, not in the v1.3.3 section.

I also don't understand what you mean by "Trial fix".  You don't know whether or not it is going to work?  If not I would still advise making the message be that it is a fix, and before merging we'll ask the user to to test it.  Also the message should better describe what kind of failure it was.